bfex/components/scrapper/run.py

```diff
@@ -0,0 +1,20 @@
+from scrapper import *
+
+# this should be done with a real service but
+# I do not know how that will work yet
+data = json.load(open(r'../../../../faculty.json'))
+
+for item in data:
+    name_from_json = item['name']
+    name_from_json = re.sub('[.]', '', name_from_json)
+    name_list = re.findall('[A-Z][^A-Z]*', name_from_json)
+    formated_name = "-".join(name_list).lower()
```

Comment:
This code (that generates names) should go under something like bfex/common/utils.py. In reality its useful everywhere, and we may reuse it.